2022-08-21 19:54:54,246 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 07:30:10,782 — ERROR — main.py:87 —      main_event_loop() — Error during central-INFLUX db insertion: could not convert string to float: ''
2022-08-22 07:30:11,383 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 07:30:11,946 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 07:30:12,735 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 07:30:13,897 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 07:30:15,884 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 07:30:19,516 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 09:59:38,755 — ERROR — main.py:87 —      main_event_loop() — Error during central-INFLUX db insertion: could not convert string to float: ''
2022-08-22 09:59:39,379 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 09:59:39,981 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 09:59:40,790 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 09:59:42,022 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 09:59:44,105 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 09:59:47,754 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:06:52,798 — ERROR — main.py:87 —      main_event_loop() — Error during central-INFLUX db insertion: could not convert string to float: ''
2022-08-22 13:06:53,388 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:06:53,961 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:06:54,732 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:06:55,963 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:06:57,972 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:07:01,608 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:49:35,758 — ERROR — main.py:87 —      main_event_loop() — Error during central-INFLUX db insertion: could not convert string to float: ''
2022-08-22 13:49:36,337 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:49:36,884 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:49:37,647 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:49:38,890 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:49:40,951 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 13:49:44,633 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 21:34:36,155 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-22 23:22:15,910 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 07:29:40,327 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 09:11:11,788 — ERROR — main.py:87 —      main_event_loop() — Error during central-INFLUX db insertion: could not convert string to float: ''
2022-08-23 09:11:12,395 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 09:11:12,984 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 09:11:13,808 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 09:11:14,995 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 09:11:16,988 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 09:11:20,611 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 09:11:27,424 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 11:43:08,023 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:03:18,762 — ERROR — main.py:87 —      main_event_loop() — Error during central-INFLUX db insertion: could not convert string to float: ''
2022-08-23 12:03:31,336 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:21:20,917 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:21:39,640 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:24:39,546 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:25:27,832 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:27:02,727 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:30:54,513 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:31:49,566 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:33:33,959 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:34:31,489 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:36:20,395 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:36:52,383 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:40:11,826 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:40:44,468 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:43:36,283 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:44:03,566 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:48:15,306 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:49:14,853 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:51:22,548 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:51:51,064 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:52:55,140 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 12:53:20,099 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:14:34,983 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:14:51,651 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:16:44,923 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:16:59,237 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:19:10,767 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:19:40,667 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:21:20,893 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:24:53,677 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:25:15,494 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:25:49,177 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:26:49,610 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:27:35,888 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:28:48,001 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:32:37,092 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:34:00,573 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:34:21,614 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:34:51,389 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:36:01,738 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:36:26,099 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:40:54,439 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 13:41:24,867 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 15:10:36,013 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-08-23 15:11:28,064 — ERROR — main.py:36 —             __init__() — network latency restarted
2022-09-02 12:14:58,664 — ERROR — main.py:36 —             __init__() — network latency restarted
